extensions/markdown-language-features/tsconfig.json

```diff
@@ -7,5 +7,6 @@
 		"src/**/*",
 		"../../src/vscode-dts/vscode.d.ts",
 		"../../src/vscode-dts/vscode.proposed.textEditorDrop.d.ts",
+		"../../src/vscode-dts/vscode.proposed.dataTransferFiles.d.ts"
 	]
 }
```

Comment:
Can't comment on it, but there's a `plain.txt` that was added at the root of the repo which is probably accidental